src/Jira/Api.php

```diff
@@ -156,13 +177,14 @@ public function setEndpoint($url)
 	 */
 	public function getFields()
 	{
-		if ( !count($this->fields) ) {
+		// Fetch fields when the method is called for the first time.
+		if ( $this->fields === null ) {
```

Comment:
I tend to use strict type comparison instead of the more magicky methods like empty() and isset(). Matter of preference I guess. (empty("0") == true or false? :D)